doc/programs.md

```diff
@@ -113,6 +113,7 @@
 - HOSTS3, the host table compiler.
 - HSNAME, displays user's HSNAME.
 - HSNDEV, HSNAME device.
+- IBMASC, read 8-bit ASCII tape file.
```

Comment:
Read an IBM tape into an ASCII file? (CTSS interchange?)

Reply:
From the name, I suppose it was meant to work with tapes from an IBM machine, but of course it works equally well with any machine that stores 8-bit characters that way.

I don't know whether CTSS used 8-bit ASCII.